tango_simlib/tests/test_sim_xmi_parser.py

```diff
@@ -194,6 +194,10 @@
     'type': PyTango.CmdArgType.DevString,
     'inherited': 'false'}
 
+EXPECTED_QUANTITIES_LIST = ['insolation', 'temperature', 'pressure', 'rainfall',
+                            'relative-humidity', 'wind-direction',
+                            'input-comms-ok', 'wind-speed', 'image1']
```

Comment:
Nice that you are using a single list now instead of repeating it.
If you don't expect it to change at runtime, as the UPPERCASE name suggests, you could make it a tuple instead of a list.